app/views/idv/doc_auth/front_image.html.erb

```diff
@@ -0,0 +1,44 @@
+<% title t('doc_auth.titles.doc_auth') %>
```

Comment:
This file got a little re-arranged, but many of the changes may be lost in the SLIM -> HTML change. I'll annotate them.